spec/hermod/xml_section_builder/string_node_spec.rb

```diff
@@ -8,9 +8,20 @@ module Hermod
       builder.string_node :name, optional: true
       builder.string_node :title, matches: /\ASir|Dame\z/, attributes: {masculine: "Male"}
       builder.string_node :required
-      builder.string_node :gender, input_mutator: (lambda do |value, attributes|
+      builder.string_node :gender, input_mutator: (proc do |value, attributes|
```

Comment:
We should probably prefer `lambda { |value, attributes, _| }` to `proc`. Shows we know it takes 3 arguments and we're ignoring one of them, rather than (ab)using `proc` to get that behaviour.

(Also see my comment above - if we did that we'd be able to revert to `lambda do |value, attribute|` here.)